app/components/authorization-code/login-redirect/login-redirect.component.js

```diff
@@ -15,7 +15,31 @@ import Controller from './login-redirect.controller';
 const LoginRedirectComponent = {
   controller: Controller,
   template: `
-    <a href="" ng-click="$ctrl.login()" data-se="login-link">Login with Okta</a>
+    <p>
```

Comment:
Talked about it with you earlier, but preferring to keep this as is to simplify the app (too many layers already for a small sample app).